projects/kit/directives/highlight/highlight.directive.ts

```diff
@@ -31,7 +34,7 @@ export class TuiHighlight implements OnChanges {
     public tuiHighlight = '';
 
     @Input()
-    public tuiHighlightColor = 'var(--tui-service-selection-background)';
+    public tuiHighlightColor = this.options.highlightColor();
```

Comment:
is it breaking changes?

Reply:
It is very useful to have an option that provides a color token for all highlight directives in the project.